spec/models/weather_spec.rb

```diff
@@ -67,14 +67,30 @@
         ]
       }
     }
-    
+
     allow_any_instance_of(DarkSkyService).to receive(:get_weather).and_return(weather_info)
-    
+
     expect(garden.weather_data).to eq(nil)
     expect(garden_2.weather_data).to eq(nil)
     Weather.get_all_weather_data
-    
+
     expect(garden.reload.weather_data).to eq(weather_info)
     expect(garden_2.reload.weather_data).to eq(weather_info)
   end
+
+  it "can return precip type", :vcr do
```

Comment:
Could add another separate test for precip icon since they are two separate methods. There is already a feature test for this, so wouldn't say it's totally necessary, but might be nice since it is a method called directly in the view.